transform/rtcalls.go

```diff
@@ -109,77 +109,64 @@ func OptimizeReflectImplements(mod llvm.Module) {
 	if implementsSignature.IsNil() {
 		return
 	}
-	interfaceMethod := mod.NamedFunction("runtime.interfaceMethod")
-	if interfaceMethod.IsNil() {
-		return
-	}
-	interfaceImplements := mod.NamedFunction("runtime.interfaceImplements")
-	if interfaceImplements.IsNil() {
-		return
-	}
 
 	builder := mod.Context().NewBuilder()
 	defer builder.Dispose()
 
 	// Get a few useful object for use later.
-	zero := llvm.ConstInt(mod.Context().Int32Type(), 0, false)
 	uintptrType := mod.Context().IntType(llvm.NewTargetData(mod.DataLayout()).PointerSize() * 8)
 
-	defer llvm.VerifyModule(mod, llvm.PrintMessageAction)
+	// Look up the (reflect.Value).Implements() method.
```

Comment:
Because it is named something like `interface:{Align:func:{}{basic:int},AssignableTo:func:{named:reflect.Type}{basic:bool},Bits:func:{}{basic:int},Comparable:func:{}{basic:bool},ConvertibleTo:func:{named:reflect.Type}{basic:bool},Elem:func:{}{named:reflect.Type},Field:func:{basic:int}{named:reflect.StructField},FieldAlign:func:{}{basic:int},Implements:func:{named:reflect.Type}{basic:bool},Key:func:{}{named:reflect.Type},Kind:func:{}{named:reflect.Kind},Len:func:{}{basic:int},Name:func:{}{basic:string},NumField:func:{}{basic:int},NumMethod:func:{}{basic:int},Size:func:{}{basic:uintptr},String:func:{}{basic:string}}.Implements$invoke` and therefore depends on the precise set of methods in src/reflect/type.go.
So yeah, we could check that name directly, but every change to the `reflect.Type` interface would also need a change here. Doable, but kind of a burden.

Reply:
Oh, I think I misunderstood what this was doing. Alright.